pkg/source/markdown.go

```diff
@@ -108,12 +114,22 @@ func (m *markdown) parse(filename string, opts Opts) (err error) {
 			count++
 		}
 
-		if count == 1 && !doxIdFound {
-			found, err := fmt.Sscanf(line, m.escape(doxIdFmt), &m.id)
+		if count == 1 || count == 2 {
+			if !doxIgnoreFound {
+				if strings.Contains(line, m.escape(doxIgnore)) {
+					m.ignore = true
+					doxIgnoreFound = true
+					return
+				}
+			}
```

Comment:
`doxIgnoreFound` and the associated check becomes extraneous since once it becomes true, the function returns.

Reply:
Moving this above the `count` condition would allow you to not have to change the condition.

Reply:
Latest commit addresses this.